crates/ef-testing/Cargo.toml

```diff
@@ -52,3 +50,4 @@ zip = { workspace = true, optional = true }
 
 [dev-dependencies]
 tracing-subscriber = { workspace = true }
+ef-testing-proc-macro = { path = "../ef-testing-proc-macro" }
```

Comment:
is it a dev deps? a deps no? since it's required not to dev on the repo, but to run it

Reply:
it's only required in the integration tests, so setting it as a dev dep works no?